ts/screens/PreferencesScreen.tsx

```diff
@@ -0,0 +1,57 @@
+import { Body, Container, Content, H1, Text, View } from "native-base";
+import * as React from "react";
+import PreferenceList from "../components/PreferencesList";
+import ScreenHeader from "../components/ScreenHeader";
+import AppHeader from "../components/ui/AppHeader";
+import I18n from "../i18n";
+import { PreferenceItem } from "../types/PreferenceItem";
+
+const preferences: ReadonlyArray<PreferenceItem> = [
```

Comment:
I'm not sure it makes sense to build the list of preference items from an array, each preference item will be linked to a different screen and also it will have to take the value of the preference from a specific attribute of the state and handle it differently - imho it makes sense to explicitly create each preference item (perhaps even create specialized preference item components)

Reply:
The screen component each preference will link could be a property of the object, just as the title, the icon and the value preview, and possibly other properties. At that moment, making leverage on an array made sense to me, in order to avoid duplicated code.